rules/table-of-contents.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const visit = require('unist-util-visit');
+const chalk = require('chalk');
+
+module.exports = (ast, file, language, done) => {
+	visit(ast, 'heading', (node, index) => {
+		const title = node.children[0].value;
+
+		if (index === 1 && title !== 'Contents') {
+			file.warn(`The first section must be ${chalk.bold('Contents')}`);
```

Comment:
No need for Chalk here. I intend to use https://github.com/sindresorhus/eslint-formatter-pretty soon, which does this for us as long as it uses this format:

```
file.warn('The first section must be `Contents`');
```